### PR TITLE
HEEDLS-194 Add download summary link to button

### PR DESCRIPTION
**Pull request by benjimarshall**
## Changes
Add progress ID and app configuration to the view model parameters to
generate the URL for the summary link.

## Testing
Update the unit tests to reflect these changes. Tested in Google Chrome, but not with the old code running (I often have issues getting the old system running with SQL requests timing out). It should be easier to test on the test site, and I doubt there will be any issues with it considering how small the change is.

### CODE REVIEW

File: DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/CourseCompletionViewModelTests.cs [hunk 1]
```diff
@@ -3,11 +3,24 @@
...
+    using FakeItEasy;
     using FluentAssertions;
+    using Microsoft.Extensions.Configuration;
     using NUnit.Framework;
 
     class CourseCompletionViewModelTests
     {
+        private IConfiguration config;
+        private const string BaseUrl = "https://example.com";
+        const int ProgressId = 23;
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/CourseCompletionViewModelTests.cs:15]: private? 🙂

**Review by FridaTveit — COMMENTED** [review]
One comment, otherwise looks good 🙂

File: DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/CourseCompletionViewModelTests.cs (hunk 1 above)
Comment (benjimarshall) [reply DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/CourseCompletionViewModelTests.cs]: Yep, sorry! Fixed in 6a31d72 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**